docs/book/src/plugins/kustomize-v2-alpha.md

```diff
@@ -1,3 +1,9 @@
+note: We bumped kustomize v5 on kustomize/v2-alpha which is used by go/v4-alpha.
+Below changes may affect your current kubernetes operator. So please take a look at it.
+Remove deprecated alpha cfg and fn commands
+Duplicate fields in Kustomization files will now result in an error
+Other than that they have fixed remote urls, Drop support for a very old, legacy style of patches, Drop support for using gh: as a host, Remove a potential build-time side-effect in ConfigMapGenerator and SecretGenerator, etc.
+For detail information kindly take a look on [this](https://github.com/kubernetes-sigs/kustomize/releases) page
 # Kustomize v2-alpha
```

Comment:
Hi @Sajiyah-Salat,

Thank you for looking on this one.  What we actually need to do is:
- Change here https://github.com/kubernetes-sigs/kubebuilder/blob/bb2eb918ccdd7797a2e7482718bebf3433b3e3e0/docs/book/src/plugins/kustomize-v2-alpha.md#when-to-use-it to say that is 5x instead of 4.x
- Update the section `When to use it` this plugin based on the changes in the release notes from 3.x to 5.x into the section: https://github.com/kubernetes-sigs/kubebuilder/blob/bb2eb918ccdd7797a2e7482718bebf3433b3e3e0/docs/book/src/plugins/kustomize-v2-alpha.md#when-to-use-it
- Add the release notes from 4.x to 5.x under the Future references section: https://github.com/kubernetes-sigs/kubebuilder/blob/bb2eb918ccdd7797a2e7482718bebf3433b3e3e0/docs/book/src/plugins/kustomize-v2-alpha.md#further-resources so that users can also check it out.

Could you please help us out with?
WDYT?

Reply:
Thanks you for this detail clarification. I will surely give it a try.